src/calendar/day/basic/index.js

```diff
@@ -41,11 +41,17 @@ class Day extends Component {
   }
 
   render() {
-    const containerStyle = [this.style.base];
-    const textStyle = [this.style.text];
+    let containerStyle = [this.style.base];
+    let textStyle = [this.style.text];
     const dotStyle = [this.style.dot];
-    
+
     let marking = this.props.marking || {};
+
+    if (marking.weekNumber) {
+      containerStyle = [this.style.weekNumberContainer];
+      textStyle = [this.style.weekNumberText];
+    }
```

Comment:
I am not sure why you need to convert to let. I think that `weekNumber` should have all the properties of `base` and extend it with `weekNumberContainer`.
So you can do:
```
containerStyle.push(this.style.weekNumberContainer);
```
Please see my comment on the stylesheet.